schemas/includes/person.json

```diff
@@ -15,6 +15,10 @@
       "type": "string",
       "description": "Name of the jurisdiction in which the entity is based"
     },
+    "company": {
```

Comment:
We certainly discussed it, but I don't think we made any decisions.  Are you suggesting that `relationship_with_company` would be a property inside a person, or inside a person's company?  I think the former makes more sense.